README.md

```diff
@@ -132,9 +132,12 @@ The `state` object should include:
 {
   title: string;
   subtitle?: string;
-  progressBar: { // Only one property, either date or progress, is available at a time as they share a single progress bar component
-    date?: number; // Set as epoch time in milliseconds. This is used as an end date in a timer.
-    progress?: number; //Set amount of progress in the progress bar
+  progressBar: { // Only one property (date, progress, or elapsedTimer) is available at a time
+    date?: number; // Set as epoch time in milliseconds. This is used as an end date in a countdown timer.
+    progress?: number; // Set amount of progress in the progress bar (0-1)
+    elapsedTimer?: { // NEW: Count UP timer (elapsed time from start)
```

Comment:
We don’t typically note a feature’s newness in its documentation. The release notes highlight what each version introduces, while the documentation describes the library’s current state.

```suggestion
    elapsedTimer?: { // Count UP timer (elapsed time from start)
```